server/controllers/UsersControler.js

```diff
@@ -0,0 +1,88 @@
+import createUserSchema from '../joi_schemas/createuser_schem';
+import Joi from '@hapi/joi';
+import users from '../model/userModel';
+import loginUserSchema from '../joi_schemas/loginUserSchema';
+
+class usersClass{
+    createUser(req,res){
+        const  schemasValidation=Joi.validate(req.body, createUserSchema);
+        if(schemasValidation.error){
+            const validationErrors=[];
+            for(let i=0; i<schemasValidation.error.details.length;i++){
+                validationErrors.push(schemasValidation.error.details[i].message.split('"').join(" "));
+            }
+            return res.status(400).json({
+                status: 400,
+                error: validationErrors[0]
+            });
+        }
+        else{
+            const Email= req.body.email;
+            const userFound= users.find(user=>user.email===Email);
+            if(userFound){
+                return res.status(400).json({
+                    status:400,
+                    error: "user with such email already exists"
+                })
+            }
+            else{
+                const newUser=req.body;
+                users.push(newUser);
+                return res.status(200).json({
+                status: 200,
+                message: "user created",
+                data: users
```

Comment:
you should return the data of the user that was created not all the users in your backend